### ISSUE TITLE
Color of hover and focus on navbar-default is not prominent

### ISSUE THREAD

**Issue by PoojaKumar** [twbs/bootstrap#17592]
On hovering/focusing on a li element of navbar-default the color change is not prominent. So I propose that a more distinguishable color should be used.

Before changing the color

![image](https://cloud.githubusercontent.com/assets/6650175/9839016/89d7210c-5a8a-11e5-8f80-4ee8ae65946f.png)

After changing the color

![image](https://cloud.githubusercontent.com/assets/6650175/9839039/eb882874-5a8a-11e5-9c00-2bd6438cb23d.png)

**Comment by cvrebert** [comment]
What version of Bootstrap are you using?

**Comment by PoojaKumar** [comment]
I have used bootstrap-3.3.5-dist

**Comment by sylvesterwillis** [comment]
I made a small change to make the hover color black instead of a dark grey and here are the differences.
Before hover:
<img width="292" alt="before hover" src="https://cloud.githubusercontent.com/assets/707556/10143781/1063d1ee-65e8-11e5-9e5a-d60bf6ebdbfd.png">

After hover:
<img width="308" alt="on hover" src="https://cloud.githubusercontent.com/assets/707556/10143786/1bd355fe-65e8-11e5-86c1-81fcac342820.png">

I'm not too sure about using another color other than black since this is meant to be the default, and black is a good color for a default that also increases the contrast of the color change on hover.

**Comment by mdo** [comment]
Given the age of v3, I don't think we should be changing this right now. I agree it's not great, but small late additions like this can be frustrating for folks who haven't touched this stuff in a long time. Closing as a won't fix for now.
